build-config/checkstyle/build.gradle

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Check in all places

Reply:
I will update

Reply:
Updated. commit id is 0d9c73ad49c7e4705e060fdf7943349bd8470692